<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cwSender</class>
 <widget class="QMainWindow" name="cwSender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>Send the text in the edit box</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="8">
     <widget class="QPlainTextEdit" name="transcriptText">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>CW Transmission Transcript</string>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="8">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Macros</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="4">
        <widget class="QPushButton" name="macro5btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="macro2btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="macro4btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 4</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QCheckBox" name="macroEditChk">
         <property name="toolTip">
          <string>Check this box to enter edit mode, where you can then press the macro buttons to edit the macros.</string>
         </property>
         <property name="text">
          <string>Edit Mode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="macro7btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 7</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="macro3btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="macro6btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="macro10btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 10</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="macro9btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="macro8btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 8</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="macro1btn">
         <property name="toolTip">
          <string>Macro Access Button</string>
         </property>
         <property name="text">
          <string>Macro 1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Seq</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sequenceSpin">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sequence number, for contests. &lt;/p&gt;&lt;p&gt;Substitute &amp;quot;%1&amp;quot; in your macro text to use it. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Break In</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="stopBtn">
      <property name="toolTip">
       <string>Stop sending CW</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="breakinCombo">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the desired break-in mode:&lt;/p&gt;&lt;p&gt;1. None: You must manually key and unkey the radio.&lt;/p&gt;&lt;p&gt;2. Semi: Transmit is automatic and switches to receive at the end of the text.&lt;/p&gt;&lt;p&gt;3. Full: Same as semi, but with breaks between characters when possible.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <item>
       <property name="text">
        <string>Off</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Full</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="textToSendEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Type here to send text as CW</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string notr="true"/>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAtTop</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>WPM:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QSpinBox" name="wpmSpin">
      <property name="toolTip">
       <string>Set the Words Per Minute</string>
      </property>
      <property name="minimum">
       <number>6</number>
      </property>
      <property name="maximum">
       <number>48</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="sendBtn">
      <property name="text">
       <string>Send</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
